templates/thank-you.html

```diff
@@ -46,24 +47,31 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
-  {% if content.view_submitted_response.enabled %}
+  {% if content.view_submitted_response.enabled and not content.view_submitted_response.expired %}
     <h1 data-qa="view-submitted-response-title">{{ _("Get a copy of your answers") }}</h1>
+    <p
+      class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
     {% if not content.view_submitted_response.expired %}
-      <p class="ons-u-mb-s">{{ _("You can <a href='{url}' id='view-submitted-response-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
-      {% call onsPanel({
-        "id": "view-submitted-response-warning",
-        "type": "warn"
-      }) %}
-        <p>{{ _("For security your answers will only be available to view for 45 minutes") }}</p>
-      {% endcall %}
-    {% else %}
-      {% call onsPanel({
+      {{ onsTimeoutPanel ({
+          "id": "view-submitted-response-countdown",
+          "minutesTextSingular": _("minute"),
+          "minutesTextPlural": _("minutes"),
+          "secondsTextSingular": _("second"),
+          "secondsTextPlural": _("seconds"),
+          "countdownText": _("For security, your answers will only be available to view for another"),
+          "nojsText": _("For security, your answers will only be available to view for another ") + content.view_submitted_response.expiry_minutes|string + _(" minutes"),
```

Comment:
Not related to Lewis comments, but can't make new comments separately. This needs to be double-checked.

- This currently wouldn't work for when < 2 minutes are left, as it would say `... for another 1 minutes` or `... for another 0 minutes`. This can be solved by using `ngettext` which can support pluralisation. It still would work for < 60 seconds left without additional logic. Worth clarifying with Donna whether we need this, or for non-js it should just say `For security your answers will only be available to view for 45 minutes`
- If the decision is to show the remaining time, then the string concatenation should use `.format()` otherwise, the extracted string will not be friendly to the translator. It should be done in the same way as: https://github.com/ONSdigital/eq-questionnaire-runner/blob/0881539fa56052d07e06f5f4d44a8007661dfd68/templates/thank-you.html#L53

Reply:
You are correct, it wont reflect the exact expiry as it will round to nearest int. i think it is not worth to put extra effort on this one. best to use hard coded 45 minutes as ticket suggests. i was thinking it would have been nice to use what the env var (`VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS`) is set to which is more likely to be `2700`. This case is also an edge case . if you have JS disabled then you can't even proceed with survey.